src/main/java/io/jenkins/plugins/appcenter/AppCenterRecorder.java

```diff
@@ -121,7 +128,8 @@ private boolean uploadToAppCenter(Run<?, ?> run, FilePath filePath, TaskListener
                 getOwnerName(),
                 getAppName(),
                 getPathToApp(),
-                getDistributionGroups()
+                getDistributionGroups(),
+                fetchReleaseNotes(run, filePath, run.getEnvironment(taskListener))
```

Comment:
For consistency this should be `getReleaseNotes()` and it should only return the appropriate release notes method.